docs/odh/oci-model-storage.md

```diff
@@ -8,15 +8,227 @@ which also explains how to deploy models from OCI images.
+tgis-grpc-serving-template           Text Generation Inference Server (TGIS) is a high performance inference engin...   0 (all set)   1
+vllm-runtime-template                vLLM is a high-throughput and memory-efficient inference and serving engine f...   0 (all set)   1
+```
+
+The template that is applicable for KServe and holds the vLLM configuration
+is the one named as `vllm-runtime-template`. To create an instance of it, run the
+following command:
+```shell
+oc process -n opendatahub -o yaml vllm-runtime-template | oc apply -f -
+```
+
+You can verify that the `ServingRuntime` has been created successfully with the
+following command:
+```shell
+oc get servingruntimes
+
+NAME           DISABLED   MODELTYPE   CONTAINERS         AGE
+vllm-runtime              vLLM        kserve-container   11s
+```
+
+Notice that the ServingRuntime has been created with `vllm-runtime` name.
+
+Now that the `ServingRuntime` is configured, the Granite model stored in an OCI image can
+be deployed by creating an `InferenceService` resource:
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: sample-isvc-using-oci
+spec:
+  predictor:
+    model:
+      runtime: vllm-runtime # This is the name of the ServingRuntime resource
+      modelFormat:
+        name: vLLM
+      storageUri: oci://quay.io/<user_name>/<repository_name>:<tag_name>
+      args:
+        - --dtype=half
+      resources:
+        limits:
+          nvidia.com/gpu: 1
+        requests:
+          nvidia.com/gpu: 1
+```
+
+> [!IMPORTANT]
+> The resulting `ServingRuntime` and `InferenceService` configurations won't set
+> any CPU and memory limits.
+
+Once the `InferenceService` resource is created, KServe will deploy the model
+stored in the OCI image referred by the `storageUri` field. Check the status
+of the deployment with the following command:
+```shell
+oc get inferenceservice
+
+NAME                    URL                                                       READY   PREV   LATEST   PREVROLLEDOUTREVISION   LATESTREADYREVISION                     AGE
+sample-isvc-using-oci   https://sample-isvc-using-oci-oci-model-example.example   True           100                              sample-isvc-using-oci-predictor-00001   2m11s
+```
+
+> [!IMPORTANT]
+> Remember that, by default, models are exposed outside the cluster and not
+> protected with authorization. Read the [authorization guide](authorization.md#deploying-a-protected-inferenceservice)
+> and the [private services guide (TODO)](#TODO) to learn how to privately deploy
+> models and how to protect them with authorization.
+
+Test the model is working:
+```sh
+# Query
+curl https://sample-isvc-using-oci-oci-model-example.apps.rosa.ehernand-test.v16g.p3.openshiftapps.com/v1/completions \
+      -H "Content-Type: application/json" \
+      -d '{
+          "model": "sample-isvc-using-oci",
+          "prompt": "¿Qué es el modelo IBM granite-3?.",
```

Comment:
Not sure about the example not being in English.
Personally I don't mind but I don't think we have ever done that in ODH or RHOAI doc.

Reply:
OK, I've put aside my MX hat and tried again in English.